recipe/build.sh

```diff
@@ -33,27 +33,41 @@ TEST_CMD="eval cd tests && make check-local && cd -"
 if [[ "$target_platform" == "linux-64" ]] || [[ "$target_platform" == "linux-32" ]] || [[ "$target_platform" == "osx-64" ]]; then
   ARCH_OPTS_SINGLE="--enable-sse --enable-sse2 --enable-avx"
```

Comment:
IDK when these flags snuck in, but I didn't think avx was allowed given our generic builds.

Reply:
They have runtime cpu detection. This enables building with sse2

Reply:
Ahh great!